### PR TITLE
Add more example for sweeper element.

### PR DESCRIPTION
**Pull request by FabienBounoir**
the use of the sweeper is not documented enough, so I made an example of how the filter works in the sweeper.

### CODE REVIEW

**Comment by jaw0r3k** [comment]
What about keeping but itself in cache?

**Comment by FabienBounoir** [comment]
> What about keeping itself in cache?
I just submitted the idea to show explicitly how to manage the cache with the filter method in the sweeper

**Comment by jaw0r3k** [comment]
But it can be better lol

**Comment by FabienBounoir** [comment]
> 

yes

File: guide/miscellaneous/cache-customization.md [hunk 1]
```diff
@@ -83,6 +83,10 @@ const client = new Client({
 			interval: 3600, // Every hour...
 			lifetime: 1800,	// Remove messages older than 30 minutes.
 		},
+		users: {
+			interval: 3600, // Every hour...
+			filter: user => user.bot, // Remove all bots.
```
Comment (megatank58) [inline guide/miscellaneous/cache-customization.md:88]:
```suggestion
			filter: user => user.bot && user.id !== client.user.id, // Remove all bots.
```

**Review by almostSouji — APPROVED** [review]

**Pull request merged**